extensions/session-time-counter/src/extension.ts

```diff
@@ -0,0 +1,41 @@
+import * as vscode from "vscode";
+
+export function activate(context: vscode.ExtensionContext) {
+  let sessionTime = 0;
+  let lastWindowFocus = Date.now();
+
+  let shouldCount: boolean = vscode.window.state.focused;
+
+  const currentSessionTimeItem = vscode.window.createStatusBarItem(
+    vscode.StatusBarAlignment.Right
+  );
+  currentSessionTimeItem.show();
+
+  const updateSesstionTime = () => {
+    if (shouldCount) {
+      const now = Date.now();
+      sessionTime += now - lastWindowFocus;
+      lastWindowFocus = now;
+    }
+  };
+
+  const interval = setInterval(() => {
+    updateSesstionTime();
+    currentSessionTimeItem.text = (sessionTime / 1000).toFixed(0) + " seconds";

```

Comment:
Let's use https://www.npmjs.com/package/ms to do this

Reply:
I've updated with https://github.com/Seneca-CDOT/vscode-seneca-college/pull/16/commits/5f9767065b2f623e76c00f7aa28e8ea59d82b4c8